src/features/landing/components/ForWhoSection/styles.ts

```
@@ -1,6 +1,48 @@
 import breakpointsTheme from '@/components/UiBreakpoints';
+import { media } from '@/components/UiBreakpoints/Theme/media';
 import colorTheme from '@/components/UiColorTheme';
 
+
+const bp: { xs: number; sm: number; md: number; lg: number; xl: number } = breakpointsTheme.breakpoints.values; 
```

ESLint

[error] 'bp' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)